x-pack/plugins/spaces/public/views/nav_control/components/spaces_menu.tsx

```diff
@@ -106,19 +106,21 @@ class SpacesMenuUI extends Component<Props, State> {
     const { intl } = this.props;
     return (
       <div key="manageSpacesSearchField" className="spcMenu__searchFieldWrapper">
-        <EuiFieldSearch
-          placeholder={intl.formatMessage({
-            id: 'xpack.spaces.navControl.spacesMenu.findSpacePlaceholder',
-            defaultMessage: 'Find a space',
-          })}
-          incremental={true}
-          // FIXME needs updated typedef
+        {
           // @ts-ignore
-          onSearch={this.onSearch}
-          onKeyDown={this.onSearchKeyDown}
-          onFocus={this.onSearchFocus}
-          compressed
-        />
+          <EuiFieldSearch
+            placeholder={intl.formatMessage({
+              id: 'xpack.spaces.navControl.spacesMenu.findSpacePlaceholder',
+              defaultMessage: 'Find a space',
+            })}
+            incremental={true}
+            // FIXME needs updated typedef
```

Comment:
We used to be able to put the `// @ts-ignore` directly above the invalid prop, which meant if we had another invalid prop we'd get a TSC error. Moving this to on top of the `EuiFieldSearch` allows the entire element to be invalid. Do you know why we can't do the approach any longer?

Reply:
unfortunately not. They has changed reporting mechanism for such errors https://github.com/Microsoft/TypeScript/issues/26423

Reply:
I'm not finding any alternatives at this point in time, but it does eliminate the targeting ignoring that we used to be able to perform, which means we really shouldn't be relying on ts-ignore in these situations. I'll open a PR to fix the type defs in EUI and get rid of these in a subsequent PR.